momepy/preprocessing.py

```diff
@@ -150,9 +150,9 @@ def preprocess(
                         geoms.append(blg[blg["mm_uid"] == j].iloc[0].geometry)
                         blg.drop(blg[blg["mm_uid"] == j].index[0], inplace=True)
                 new_geom = shapely.ops.unary_union(geoms)
-                blg.loc[
-                    blg.loc[blg["mm_uid"] == key].index[0], blg.geometry.name
-                ] = new_geom
+                blg.loc[blg.loc[blg["mm_uid"] == key].index[0], blg.geometry.name] = (
```

Comment:
Is this extra indexing needed due to an update in Pandas/GeoPandas?

Reply:
haven't touched that one (except for running black on it) i'll leave the question to @martinfleis